lib/Doctrine/Table.php

```diff
@@ -1725,7 +1725,7 @@ public function findBy($fieldName, $value, $hydrationMode = null)
      * @param string $column            field for the WHERE clause
      * @param string|array $value       prepared statement parameter
      * @param int $hydrationMode        Doctrine_Core::HYDRATE_ARRAY or Doctrine_Core::HYDRATE_RECORD
-     * @return Doctrine_Record
+     * @return Doctrine_Record|false
```

Comment:
According to  `Query::fetchOne()` the return could be `array|Doctrine_Record|false`: those whould be added too, wdyt?

At the same time, the implementation of `Query::fetchOne()` has:
```php
        if (is_scalar($collection)) {
            return $collection;
        }
```

Which then, could return a scalar too `int|bool|float`